tests/unit/rpc/RPCHelpersTests.cpp

```diff
@@ -58,7 +58,6 @@
 #include <cstddef>
 #include <cstdint>
 #include <optional>
-#include <sstream>
```

Comment:
clang-tidy was triggered for my ci (due to config change), so I had to fix this here